components/tools/ToolDashboard.js

```diff
@@ -48,6 +48,18 @@ export default function ToolDashboard() {
     }, 1000);
   }, []);
 
+  // useEffect to scroll particular div into view
+  useEffect(() => {
+    setTimeout(() => {
+      const path = router?.asPath
+      const id = path?.slice(path?.indexOf("#") + 1).replace("%20", " ")
+      const element = document.getElementById(id)
```

Comment:
Instead of using `id` attribute, can't we use `ref` here to add the behavior?

Reply:
We usually access the DOM elements directly in react. React has it's own virtual DOM feature which enables better performance on the website. That's why I asked to use ref in this case. We used `id` just to reference the category and link it in dropdown.